dvc/remote/http.py

```diff
@@ -45,7 +47,7 @@ def _download(self, from_info, to_file, name=None, no_progress_bar=False):
             disable=no_progress_bar,
         ) as pbar:
             with open(to_file, "wb") as fd:
-                for chunk in request.iter_content(chunk_size=self.CHUNK_SIZE):
+                for chunk in response.iter_content(chunk_size=self.CHUNK_SIZE):
                     fd.write(chunk)
                     fd.flush()
```

Comment:
I don't think we need flush. It is probably a remnant from the time we did http resume.